docs/reference/commandline/build.md

```diff
@@ -209,12 +207,12 @@ Removing intermediate container 9c9e81692ae9
 Removing intermediate container 02071fceb21b
 ```
 
-This example specifies that the `PATH` is `.`, and so all the files in the
-local directory get `tar`d and sent to the Docker daemon. The `PATH` specifies
+This example specifies that the `PATH` is `.`, and so `tar`s all the files in the
+local directory and sends them to the Docker daemon. The `PATH` specifies
 where to find the files for the "context" of the build on the Docker daemon.
 Remember that the daemon could be running on a remote machine and that no
 parsing of the Dockerfile happens at the client side (where you're running
-`docker build`). That means that *all* the files at `PATH` get sent, not just
+`docker build`). That means that *all* the files at `PATH` are sent, not just
```

Comment:
We still (separate from this PR) need to look at some of this; when using BuildKit, the daemon uses an session between the CLI and the daemon, and only requests files from the CLI / build context that it needs for the build (unlike the classic builder, which just tars the whole build-context)